interfaces/src/subscriptions.ts

```diff
@@ -1,5 +1,7 @@
 import { Operation } from 'fast-json-patch';
 
+export type Version = [number, number];
```

Comment:
Why is this redefined here?
Seems like we should simply unify the `subscriptions` and `db` sources.

Reply:
Just reimporting for now.

Unifying the 2 files can come later.